README.md

```diff
@@ -1,55 +1,80 @@
-
-The design is informed by discussions, talks and countless hours of hard and thoughtful work invested into many previous systems, [frameworks](https://github.com/NEventStore), [samples](https://github.com/thinkbeforecoding/FsUno.Prod), [forks of samples](https://github.com/bartelink/FunDomain), the outstanding continuous work of the [EventStore](https://github.com/eventstore) founders and team and the wider [DDD-CQRS-ES](https://groups.google.com/forum/#!forum/dddcqrs) community. It would be unfair to single out even a small number of people despite the immense credit that is due.
+A unified programming model for event-sourced processing against stream-based stores including snapshots, caching and other state management/retrieval optimizations. The library does not and will not handle projections, subscriptions etc. (see [Propulsion](https://github.com/jet/propulsion) for that).
+
+Equinox is focused on remaining a set of low dependency _libraries_ that _you_ compose into an architecture that fits your apps needs. This means that Equinox can also help with general accesss strategies for your storage options. The library is neither a source for Best Practice Architecture/object model/processing pipeline nor is it imposing a one-size-fits-all model on you. You decide what facilities make sense for your context and Equinox covers the infrastructural aspects. If you are new to event-sorucing we do have sample patterns that are located in [`samples`](/samples) or [templates](https://github.com/jet/dotnet-templates). This core repo will never feature tight integration with any specific web or services framework.
+
+# Table of Contents
+* [Quickstart](#quickstart)
+* [Design Motivation](#design-motivation)
+* [Features](#features)
+* [Currently Supported Data Stores](#currently-supported-data-stores)
+* [Components](#components)
+  * [Core libary](#core-library)
+  * [Serialization Support](#serialization-support)
+  * [Data Store Libraries](#data-store-libraries)
+  * [Projection Libraries](#projection-libraries)
+  * [Benchmarking Tool](#benchmarking-tool)
+  * [Starter Project Templates and Sample Applications](#starter-project-templates-and-sample-applications)
+* [Overview](#overview)
+* [Templates](#templates)
+* [Samples](#samples)
+* [Building](#building)
+* [Releasing](#releasing)
+* [FAQ](#faq)
+* [Acknowledgements](#acknowledgements)
+* [Further Reading](#further-reading)
+
+# Quickstart
```

Comment:
there's quite a few links to `#quickstart` externally - maybe rename to _Getting Started_?